content/page/faq.md

```diff
@@ -41,3 +38,60 @@ currently not supported and should be done by the firmware instead.
 Initially, all three of these projects were called NERF. Because this would have
 only become more confusing, and because we do not want to be prescriptive of the
 initramfs, we named the Linux kernel in firmware *LinuxBoot*.
+
+### Will Anyone Ever Ship This?
+
+[Horizon Computing](http://www.horizon-computing.com) plans to ship Open
+Compute Platform
+[Winterfell](http://www.horizon-computing.com/?project=winterfell) nodes with
+LinuxBoot/NERF in Q2 2018.
+
+### Do I Need A coreboot-compatible Motherboard?
+
+No, LinuxBoot is compatible with UEFI as well as coreboot. You can use
+LinuxBoot as a UEFI DXE or as the coreboot ramstage. See Ron Minnich's talk
```

Comment:
Change coreboot ramstage to coreboot payload and add this link here https://review.coreboot.org/c/coreboot/+/23071 -> Currently there is no ramstage replacement

Reply:
Link can be added as part of the two words "coreboot payload"

Reply:
Done.